webapi/Models/Request/DocumentImportForm.cs

```diff
@@ -50,4 +50,9 @@ public enum DocumentScopes
     /// Will be used to create the chat message representing the document upload.
     /// </summary>
     public string UserName { get; set; } = string.Empty;
+
+    /// <summary>
+    /// Flag indicating whether user has content safety enabled from the client.
+    /// </summary>
+    public bool UseContentSafety { get; set; } = false;
```

Comment:
Is this setting exposed in the front-end on a per-document basis or globally?

Reply:
Globally - it's in the settings dialog. It's disabled by default if it's not enabled in the backend.
<img width="718" alt="Screenshot 2023-08-17 at 11 57 42 AM" src="https://github.com/microsoft/chat-copilot/assets/125500434/7218bc47-06fe-47a0-8e35-96cccff5926b">